pkg/storage/system_intake.go

```diff
@@ -260,7 +260,8 @@ func (s *Store) UpdateSystemIntakeNP(ctx context.Context, np sqlutils.NamedPrepa
 			cedar_system_id = :cedar_system_id,
 			has_ui_changes = :has_ui_changes,
 			trb_follow_up_recommendation = :trb_follow_up_recommendation,
-			contract_name = :contract_name
+			contract_name = :contract_name,
+			system_relation_type = :system_relation_type
```

Comment:
I think you'll also need to update the `Fetch` method in here to explicitly return this new field

Reply:
Nevermind -- got something mixed up when reviewing!